src/components/cards/PairPriceGraph/PairPriceGraph.vue

```diff
@@ -21,17 +21,33 @@ import { getAddress } from '@ethersproject/address';
 import QUERY_KEYS from '@/constants/queryKeys';
 import useWeb3 from '@/services/web3/useWeb3';
 
+/**
+ *
+ * @param inputAsset The address of the input asset
+ * @param outputAsset The address of the output asset
+ * @param nativeAsset The address of the native asset on the network
+ * @param wrappedNativeAsset The address of the wrapped native asset on the network
+ * @param days The number of days to pull historical data for
+ * @param inverse Swaps the pricing calc to be output/input rather than input/output
+ */
 async function getPairPriceData(
   inputAsset: string,
   outputAsset: string,
   nativeAsset: string,
   wrappedNativeAsset: string,
-  days: number
+  days: number,
+  inverse?: boolean
 ) {
   let _inputAsset =
     inputAsset === nativeAsset ? wrappedNativeAsset : inputAsset;
   let _outputAsset =
     outputAsset === nativeAsset ? wrappedNativeAsset : outputAsset;
+
+  if (inverse) {
+    const tempInputAsset = _inputAsset;
```

Comment:
could you try `[_inputAsset, _outputAsset] = [_outputAsset, _inputAsset];`

Reply:
oh wow, didn't know you could do that